### PR TITLE
Warn about imported appointments on the edit page

### PR DESCRIPTION
**Pull request by AndrewVos**
image

### CODE REVIEW

**Comment by benlovell** [comment]
Nicely done. 💯

File: app/views/appointments/edit.html.erb [hunk 1]
```diff
@@ -3,6 +3,18 @@
...
 ) %>
 
+<% if @appointment.imported? %>
+  <div class="alert alert-warning t-appointment-was-imported-message" role="alert">
+    <p>
+      This appointment was automatically imported from Booking Bug,
+      which means that the date of birth is not valid.
+    </p>
+    <p>
+      Do not read the date of birth out to the customer.
```
Comment (benbarnett) [inline app/views/appointments/edit.html.erb]: Such firmness 😤😡😡 

Maybe "Please refer to the date of birth in the notes field" or something?

**Comment by Guntrisoft** [comment]
Beautiful 😍

File: app/views/appointments/edit.html.erb (hunk 1 above)
Comment (AndrewVos) [inline app/views/appointments/edit.html.erb]: :) done

File: app/views/appointments/edit.html.erb [hunk 2]
```diff
@@ -3,6 +3,18 @@
...
 ) %>
 
+<% if @appointment.imported? %>
+  <div class="alert alert-warning t-appointment-was-imported-message" role="alert">
+    <p>
+      This appointment was automatically imported from Booking Bug,
+      which means that the date of birth is not valid.
+    </p>
+    <p>
+      Please refer to the date of birth in the notes field
```
Comment (benbarnett) [inline app/views/appointments/edit.html.erb]: Does not matter but we should probably have a full stop 💃

Comment (AndrewVos) [inline app/views/appointments/edit.html.erb]: :)

**Pull request merged**